src/XliffTasks.Tests/XliffDocumentTests.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+
+namespace XliffTasks.Tests
+{
+    public class XliffDocumentTests
+    {
+        [Fact]
+        public void NewDocument()
```

Comment:
I agere and I will fix it in a follow up PR. The tests so far are quick and dirty and I intend to circle back over them once I have end-to-end proof-of-concept in SDK repo.